app/models/workers_scaling/algorithms/resources_usage_minimization.rb

```diff
@@ -68,53 +67,53 @@ def time_constraint_check(predicted, left)
     ##
     # Increases computational power to match required system throughput
     # Does nothing if there are starting Workers already
-    # Uses known infrastructures with highest throughput if limits are not reached
-    # If limits in all known infrastructures are reached, uses random unknown infrastructure
+    # Uses known configurations with highest throughput if limits are not reached
+    # If limits in all known configurations are reached, uses random unknown configuration
     def increase_computational_power
-      LOGGER.debug 'Need to increase computational power'
+      log(:debug, 'Need to increase computational power')
```

Comment:
Jeżeli jesteśmy już w temacie logów na poziomie debug, Scalarm produkuje ich bardzo dużo. Mam plik gdzie przy jednym działającym eksperymencie w około 7 minut zostało wyprodukowane 20 MB. Sporo z tych logów to raczej niewiele wnoszące informacje o aktualizacji paska postępu, Na około 182 tys linii logu około 70 tys jest w formacie 
```
D, [2015-12-15T14:43:21.140771 #18744] DEBUG -- : Bar last update - 2015-12-15 14:43:20 +0100
```
a kolejne 70 tys. w formacie 
```
D, [2015-12-15T14:42:15.140689 #18744] DEBUG -- : update_bar_state(1)
```
Po usunięciu tego typu linijek rozmiar spadł do 8MB.